_includes/header.html

```diff
@@ -1,16 +1,32 @@
-<header class="pldoc-header-site" role="banner" aria-labelledby="pldoc-header-site-title">
-    {% if page.title == "Home" %}
-    <h1 class="pldoc-header-site-title" id="pldoc-header-site-title">
-        <a class="pldoc-link title-name" href="{{ site.baseurl }}">
-            {{ site.title }}
-        </a>
-    </h1>
-    {% else %}
-    <h2 class="pldoc-header-site-title" id="pldoc-header-site-title">
-        <a class="pldoc-link title-name" href="{{ site.baseurl }}">
-            {{ site.title }}
-        </a>
-    </h2>
-    {% endif %}
-    <span class="pldoc-title-description copy-base">{{ site.tagline }}</span>
-</header>
+<div class="pldoc-wrapper-header-site">
+    <header class="pldoc-header-site" role="banner" aria-labelledby="pldoc-header-site-title">
+        <div class="layout-1t2t layout">
+            <main class="layout-col layout-col-a">
+                <h2 class="hd-2 de-emphasized pldoc-header-site-title">
+                    <a href="{{ site.baseurl }}">
+                        <img src="{{ site.url }}/public/images/logo-edx.svg" alt="edX logo"
```

Comment:
The alt text should be provide context re. the link here, something like 'edX UX Pattern Library home'